apps/dotcms-ui/src/app/view/components/dot-toolbar/dot-toolbar.component.ts

```diff
@@ -29,6 +34,11 @@ export class DotToolbarComponent implements OnInit {
                 });
             }
         });
+
+        this.dotNavLogoService.setInitialLogo();
```

Comment:
tests?

Reply:
We set the colors in the very begging of the app [here](https://github.com/dotCMS/core-web/blob/ng12-update/apps/dotcms-ui/src/app/app.component.ts#L24). We should set initial the logo there as well.